lib/osf-components/addon/components/form-controls/radio-button-group/radio-button/component.ts

```diff
@@ -0,0 +1,46 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { BufferedChangeset } from 'ember-changeset/types';
+
+import { RadioButtonOption } from 'osf-components/components/form-controls/radio-button-group/component';
+
+/**
+ * The Radio Button Args
+ */
+interface RadioButtonArgs{
+    option: string | RadioButtonOption;
+    valuePath: string;
+    changeset: BufferedChangeset;
+    disabled: boolean;
+    onchange: (_: string | number | boolean) => void;
```

Comment:
Similar to the above comment, can we add an optional `helpTextMapping: any` param here?